### ISSUE TITLE
Storefront a11y e2e tests fail because of `aria-required-children` rule

### ISSUE THREAD

**Issue by denyo** [porsche-design-system/porsche-design-system#2193]
Should be merged with https://github.com/porsche-design-system/porsche-design-system/issues/2184

Scope
---
With `axe-core` version 4.5.0 (2022-10-17) the rule `aria-required-children` was changed to fail for children which are not listed as required.  
This change causes basically every storefront page to fail. It appears the root cause is inside `p-scroller`.

```
aria-required-children (critical) on light:
– p-tabs-bar[size="medium"] >>> p-scroller
– .playground[data-v-8dbc1b2a=""]:nth-child(7) > p-tabs-bar >>> p-scroller
– .playground[data-v-8dbc1b2a=""]:nth-child(7) > .example.example--light[data-v-8dbc1b2a=""] > .code-block.code-block--light.code-block--default > p-tabs-bar[theme="light"] >>> p-scroller
– .playground[data-v-8dbc1b2a=""]:nth-child(10) > p-tabs-bar >>> p-scroller
– .playground[data-v-8dbc1b2a=""]:nth-child(10) > .example.example--light[data-v-8dbc1b2a=""] > .code-block.code-block--light.code-block--default > p-tabs-bar[theme="light"] >>> p-scroller
```
To workaround this, failing `aria-required-children` checks are currently ignored.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
- https://github.com/dequelabs/axe-core/blob/develop/CHANGELOG.md#450-2022-10-17
- https://github.com/porsche-design-system/porsche-design-system/actions/runs/3572031954/jobs/6005143892
- See also issue #2184 

Acceptance criteria
---
- [ ] Criteria

Subtasks
---
- [ ] Task
